libs/base/System.idr

```diff
@@ -98,6 +98,22 @@ prim__setEnv : String -> String -> Int -> PrimIO Int
          supportNode "unsetEnv"
 prim__unsetEnv : String -> PrimIO Int
 
+%foreign "C:idris2_enableRawMode, libidris2_support, idris_support.h"
+prim__enableRawMode : (1 x : %World) -> IORes ()
+
+%foreign "C:idris2_disableRawMode, libidris2_support, idris_support.h"
+prim__disableRawMode : (1 x : %World) -> IORes ()
+
+||| Enable raw mode for stdin (read unbuffered with no echo)
```

Comment:
Regardless of errors or no-Ops, I suggest we add the lack of windows support to the code docs.